### PR TITLE
rust-agent: Update README

### PR DESCRIPTION
**Pull request by yyyeerbo**
rust agent does not use grpc as submodule for a while, update README
to reflect the change.

Fixes: #196

Signed-off-by: Yang Bo <bo@hyper.sh>

### CODE REVIEW

File: src/agent/README.md [hunk 1]
```diff
@@ -43,10 +43,9 @@ The `rust-agent` depends on [`grpc-rs`](https://github.com/pingcap/grpc-rs) by P
 - https://github.com/alipay/grpc-rs/tree/rust_agent
 
 ### Build from Source
-The rust-agent need to be built with rust nightly, and static linked with musl.
+The rust-agent need to be built with rust newer than 1.39, and static linked with musl.
```
Comment (jodh-intel) [inline src/agent/README.md:46]: Is this change related to the gRPC change below? If not, strictly it should be in a different commit with some details as to why the new minimum version is 1.39.

**Comment by jodh-intel** [comment]
Hi @yyyeerbo - there is one outstanding comment on this PR.

Also, Travis is failing:

```
ERROR: Document src/agent/README.md is not referenced
```

That error is correct. What I suggest you do is update the top-level README [1] as below to resolve this error:

```diff
diff --git a/README.md b/README.md
index 195ab2b4..5c2def1d 100644
--- a/README.md
+++ b/README.md
@@ -66,7 +66,7 @@ the first place to go if you want to use or contribute to the project.

 ##### Agent

-The [`kata-agent`](https://github.com/kata-containers/agent) runs inside the
+The [`kata-agent`](src/agent) runs inside the
 virtual machine and sets up the container environment.

 ##### KSM throttler
```

---

[1] - either on the existing commit, or as a separate commit *before* your current commit.

**Comment by grahamwhaley** [comment]
nudge @yyyeerbo - are you planning to update this at some point wrt the feedback from @jodh-intel ? thx!

**Comment by grahamwhaley** [comment]
2nd nudge @yyyeerbo ;-)

**Comment by bergwolf** [comment]
ping @yyyeerbo

**Comment by bergwolf** [comment]
travis failed with
```
ERROR: Invalid URL 'https://github.com/kata-containers/kata-agent' found in the following files:
src/agent/README.md
```

**Comment by cmaf** [comment]
ping @yyyeerbo, would you be able to address @jodh-intel's comments?

**Review by bergwolf — APPROVED** [review]

**Comment by bergwolf** [comment]
Somehow I cannot trigger these required tests.

Let's close it and rebase and handle in #833

image
